templates/download/nvidia-jetson/tab-2-jetson-orin-nano.html

```diff
@@ -0,0 +1,58 @@
+<div tabindex="0" role="tabpanel" id="jetson-orin-nano-tab" aria-labelledby="jetson-orin-nano" aria-hidden="true">
+  <div class="row p-strip is-shallow u-no-padding--top">
+    <div class="col-3">
+      <h2 class="u-hide--small">Jetson Orin Nano</h2>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0b13a01c-c9301d06-Jetson_product_tiles_Module_pages-Jetson_AGX_Orin-ftont-v002.jpg",
+        alt="",
+        width="1724",
+        height="1984",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9">
+    <div class="col-3">
+      <h3 class="p-heading--5 u-no-padding--top">Ubuntu Server 22.04</h3>
+    </div>
+    <div class="col-6">
+      <p class="u-no-padding--top">The version of optimised Ubuntu Server 22.04 is beta for now, the certified version is coming soon.</p>
+      <hr class="u-no-margin--bottom">
+      <div class="p-notification--information is-borderless">
+        <div class="p-notification__content">
+          <p class="p-notification__message">Please download boot firmware before you start your installation.</p>
+        </div>
+      </div>
+      <p>
+				<a class="p-button--positive" href="">Download <span class="u-off-screen">Ubuntu Server</span> 22.04</a>
+      </p>
+      <p><a class="p-button" href="">Download Jetson Orin Nano boot firmware</a></p>
+    </div>
+  </div>
+  <div class="row p-strip u-no-padding">
```

Comment:
```suggestion
  <div class="row">
```
These cancel each other out